pkg/controllers/service/service_lb_controller.go

```diff
@@ -55,24 +61,31 @@ func (r *ServiceLbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}()
 
 	if err := r.Client.Get(ctx, req.NamespacedName, service); err != nil {
+		if client.IgnoreNotFound(err) == nil {
+			log.Info("Not found LB service", "req", req.NamespacedName)
+			return ResultNormal, client.IgnoreNotFound(err)
+		}
 		log.Error(err, "Failed to fetch LB service", "req", req.NamespacedName)
-		return ResultNormal, client.IgnoreNotFound(err)
+		return common.ResultRequeue, err
 	}
 
 	if service.Spec.Type == v1.ServiceTypeLoadBalancer {
-		log.Info("Reconciling LB service", "LBService", req.NamespacedName)
+		log.Info("Reconciling LB service", "LBService", req.NamespacedName, "version", service.ResourceVersion, "status", service.Status)
 		metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerSyncTotal, MetricResType)
 
 		if service.ObjectMeta.DeletionTimestamp.IsZero() {
 			metrics.CounterInc(r.Service.NSXConfig, metrics.ControllerUpdateTotal, MetricResType)
-			updateSuccess(r, ctx, service)
+			err := updateSuccess(r, ctx, service)
+			if err != nil {
+				return ResultRequeue, err
+			}
 		}
 	}
 
 	return ResultNormal, nil
 }
 
-func (r *ServiceLbReconciler) setServiceLbStatus(ctx context.Context, lbService *v1.Service) {
+func (r *ServiceLbReconciler) setServiceLbStatus(ctx context.Context, lbService *v1.Service) error {
 	ipMode := v1.LoadBalancerIPModeProxy
```

Comment:
ipMode -> ipModeProxy

Reply:
it seems that ipMode is better. The value may be VIP